tensorflow/core/ops/array_ops.cc

```diff
@@ -1323,6 +1323,16 @@ REGISTER_OP("Transpose")
     .Attr("Tperm: {int32, int64} = DT_INT32")
     .SetShapeFn(TransposeShapeFn);
 
+#ifdef INTEL_MKL
+REGISTER_OP("_MklTranspose")
+    .Input("x: T")
+    .Input("perm: Tperm")
+    .Output("y: T")
+    .Attr("T: type")
+    .Attr("Tperm: {int32, int64} = DT_INT32")
+    .SetShapeFn(TransposeShapeFn);
+#endif // INTEL_MKL
```

Comment:
Nit: It has to be 2 spaces. Same everywhere else.
```suggestion
#endif  // INTEL_MKL
```